src/components/src/with-wizard-screen/style.scss

```diff
@@ -7,10 +7,10 @@
 
 .newspack-wizard {
 	// Color
-	--wp-admin-theme-color: #{colors.$primary-500};
-	--wp-admin-theme-color--rgb: #{colors.$primary-500--rgb};
-	--wp-admin-theme-color-darker-10: #{colors.$primary-600};
-	--wp-admin-theme-color-darker-10--rgb: #{colors.$primary-600--rgb};
+	--wp-admin-theme-color: var(--newspack-ui-color-primary);
+	--wp-admin-theme-color--rgb: var(--newspack-ui-color-primary-rgb);
+	--wp-admin-theme-color-darker-10: var(--newspack-ui-color-primary);
+	--wp-admin-theme-color-darker-10--rgb: var(--newspack-ui-color-primary-rgb);
 	--wp-admin-theme-color-darker-20: #{colors.$primary-700};
 	--wp-admin-theme-color-darker-20--rgb: #{colors.$primary-700--rgb};
```

Comment:
I think here we need to introduce 2 primary variations:

* primary-darker-10: #1522AF (21 34 175)
* primary-darker-20: #0B0B8D (11 11 141)

Reply:
Same as https://github.com/Automattic/newspack-plugin/pull/3621#discussion_r1892726226

Reply:
See https://github.com/Automattic/newspack-plugin/pull/3621#discussion_r1893745080